### ISSUE TITLE
Showing the coordinates for tiles in the map

### ISSUE THREAD

**Issue by kimti** [bjorn/tiled#1125]
Showing the coordinates for tiles in the map

**Comment by bjorn** [comment]
This is actually a feature that is present in the old Java version of Tiled. I think the main purpose would be to explain the coordinate system, which I'd probably prefer to do in a manual. The tile coordinates can already be inspected in Tiled by hovering and looking at the status bar.

**Comment by kimti** [comment]
i think has a Choice to show all coordinate,

**Comment by bjorn** [comment]
Yes, the old Java version of Tiled could do that. Yet, I thought it looked somewhat messy and I think you only need it very rarely, so I would prefer to put it in the documentation or possible on a grid drawn in the New Map dialog.

Or do you actually have a need for this beyond just getting an initial understanding of the coordinate system?

**Comment by kimti** [comment]
yes , i think it is very useful,  it can show me to compare between client and server

**Comment by kimti** [comment]
so if it looked somewhat messy, but only a option, i can disable or enable it.
